src/pages/forget-password/reset.js

```diff
@@ -0,0 +1,156 @@
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { useState, useEffect, useContext } from 'react'
+import axios from 'axios'
+
+import { Card, CardContent, Button, Grid, Box, IconButton, InputAdornment } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import * as yup from 'yup'
+import BlankLayout from 'src/@core/layouts/BlankLayout'
+import { ControlContext } from 'src/providers/ControlContext'
+import ResetPassForm from './forms/ResetPassForm'
+import { useWindow } from 'src/windows'
+import { useAuth } from 'src/hooks/useAuth'
+import { useForm } from 'src/hooks/form.js'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+
+const Reset = () => {
+  const theme = useTheme()
+  const { platformLabels } = useContext(ControlContext)
+  const [accountId, setAccountId] = useState({})
+  const { stack } = useWindow()
+  const auth = useAuth()
+
+  useEffect(() => {
+    setAccountId(auth?.getAC?.data?.record?.accountId)
```

Comment:
can we remove account state, useEffect and replace instead
const accountId = auth?.getAC?.data?.record?.accountId  ???

Reply:
right fixed